index.html

```diff
@@ -14,7 +14,7 @@
 
     <title>Vite + Preact + TS</title>
   </head>
-  <body>
+  <body id="parts-kit">
```

Comment:
Could we put this at the root node of [app.tsx](https://github.com/vigetlabs/parts-kit/blob/main/src/app.tsx#L121)?

Ideally I'd like for users of the JS snippet to have as few implementation steps as possible.

Reply:
ah yes, definitely.

Moved it into the `app.tsx`.